Enhanced navbar with basic styling and <Link> instead of <a> tags. Removed logo asset from public folder.

xuemak opened the pull request:
Updated navbar to have basic styling with background color, padding, and soon-to-be nav links arranged in a row. I also added an image of the KOZAN logo in the public folder for later use.

**Before:**
<img width="1378" alt="Screenshot 2025-04-09 at 11 39 59 AM" src="https://github.com/user-attachments/assets/a27d56cd-b729-40fa-825c-dd51bbb6c752" />

After:
<img width="1378" alt="Screenshot 2025-04-09 at 11 41 08 AM" src="https://github.com/user-attachments/assets/a513fd4b-9fb3-4878-ac1e-8e3f68cd349d" />

workingasintendead commented on src/components/Navbar.tsx:
```diff
@@ -2,15 +2,37 @@ import React from "react";
+            <div>
+                <ul className="flex p-0 font-medium bg-purple-200 space-x-8">
+                    <li>
+                        <a href="#">Milk Tea</a>
```
You should look into using Link instead of an <a href>
and <Image> when you get to that.

https://nextjs.org/docs/pages/api-reference/components/link

xuemak replied:
Done!

xuemak commented:
Update #2: Replaced <a> tags with <Link> per @workingasintendead's feedback for improved page loading. I also removed the KOZAN logo asset from public folder since I'm not using it for anything at this stage.
